app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,54 @@
+class InvitesController < ApplicationController
+  skip_before_action :authenticate!, only: :accept
+
+  def create
+    begin
+      unless Invite.where('email = ? AND website_id = ? AND invited_by_id =?', invite_params[:email], current_website.id, current_user.id).blank?
```

Comment:
do it like this:
```
raise 'User already invited' unless current_website.invites.where(email: invite_params[:email]).blank?
```

As we don't need to do the full check including the invited_by_id that is redundant.